babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -1470,6 +1478,66 @@ SqlTableAttribute AlterTableAttributeOnCommit() :
     { return new SqlAlterTableAttributeOnCommit(getPos(), onCommitType); }
 }
 
+/**
+  * Parses a list of alter options (ex. ADD, DROP, RENAME) for
+  * ALTER TABLE queries.
+  */
+List<SqlAlterTableOption> AlterTableOptions() :
+{
+    final List<SqlAlterTableOption> alterTableOptions =
+        new ArrayList<SqlAlterTableOption>();
+    SqlAlterTableOption alterTableOption;
+}
+{
+    alterTableOption = AlterTableOption()
+    { alterTableOptions.add(alterTableOption); }
+    (
+        <COMMA>
+        alterTableOption = AlterTableOption()
+        { alterTableOptions.add(alterTableOption); }
+    )*
+    { return alterTableOptions; }
+}
+
+/**
+  * Parses a single alter option (ex. ADD, DROP, RENAME) for
+  * ALTER TABLE queries.
+  * Used by {@code AlterTableOptions}.
+  */
+SqlAlterTableOption AlterTableOption() :
+{
+    final SqlAlterTableOption option;
+}
+{
+    (
```

Comment:
That is where the other options for RENAME, DROP will go. I suppose I could have just added it in the next PR, but I figured since I know there will be other things there, I will just do it in advance.